packages/core/src/components/SearchBox/types.ts

```diff
@@ -18,8 +19,12 @@ export interface SearchBoxProps extends Omit<HTMLProps<HTMLInputElement>, 'size'
     size?: Size;
     /** Option for search results, in form of label and value */
     options?: Option[];
-    /** Enables the expand icon */
-    showExpandIcon?: boolean;
+    /** React component for additional filtering */
+    customSearchFilter?: ReactElement<any>;
+    /** Does the component take up all available width */
+    fullWidth?: boolean;
+    /** Min width in rem/% (1rem = 10px)*/
```

Comment:
Is there a reason we're specifying `rem` here? Looks like other width props accept any unit

Reply:
Although all the unit works but we should use single unit across the app and hence suggesting rem over here.